Updates

Webkadabra opened the pull request.

Commit:
Update package name so it's available via composer

Commit:
Add `getDocumentPrice` method for `InternetDocument` model

Commit:
Add installation instructions and update config instructions

Commit:
Add `getArea` method

Commit:
`MiddleName` is not required to create `Counterparty`

Commit:
Add API methods to create & update `ContactPerson`

Commit:
Complete `saveDocument` method

Commit:
Fix rules

Commit:
Add counter party types

joni-jones commented:
Some points:
1. Your PR replaces original package name.
2. Readme changes are incorrect.
3. https://github.com/joni-jones/yii2-novaposhta/pull/2/commits/9112782416f0b02f6be7ae8c6bb8ea45752be4f7 - the `MiddleName` is required field https://devcenter.novaposhta.ua/docs/services/557eb8c8a0fe4f02fc455b2d/operations/557ebbd3a0fe4f02fc455b2e
4. Some code not PSR compatible.
5. Code not covered by tests.
6. The `call` method doesn't support the second argument. Did you test it? https://github.com/Webkadabra/yii2-novaposhta/blob/3521da7aa03976d9e638faa51ac3f152a2f2789b/src/InternetDocument.php#L87
7. Don't need to map properties into array https://github.com/Webkadabra/yii2-novaposhta/blob/3521da7aa03976d9e638faa51ac3f152a2f2789b/src/InternetDocument.php#L67.

Webkadabra commented:
@joni-jones I think I had to change package name because I could not make my repo available for composer otherwise. Now, I'm sure I was just too quick to take an easy route, so I will check this and fix package name to be of original.

I will review the rest of your comments. Also, regarding #3 - MiddleName is not required (I'm able to create records via API without `MiddleName`; more so, in my project we don't even ask customers for a middle name.

Commit:
Add `BackwardDeliveryData` and `OptionsSeat` properties

Commit:
Add `AdditionalInformation`

Commit:
Add `getDocumentDeliveryDate` method

GlitchGrp committed:
Add `InfoRegClientBarcodes` parameter

This parameter represents internal order number and if for customer
informaiton only.

GlitchGrp committed:
Add `TrackingDocument` API model

vldmr-k committed:
Update composer.json

vldmr-k committed:
Make future to rewrite RequestFactory to work with multyply Novaposhta Account

vldmr-k committed:
change methid setRequestFactory

vldmr-k committed:
Добавил возможность удаление InternetDocument.
Метод  в исходном пакете, не работал

Webkadabra committed:
Update Guzzle package dependency

Commit:
Update composer.json

Commit:
Update composer.json

Webkadabra committed:
Drop lock file

Webkadabra committed:
Merge branch 'master' of https://github.com/Webkadabra/yii2-novaposhta

Webkadabra committed:
Fix package name

Webkadabra committed:
Fix package name typo

Webkadabra committed:
build: allow guzzle 7
